test/specs/components/action.spec.jsx

```diff
@@ -271,20 +324,32 @@ xdescribe('Action', () => {
 
         describe('user has email', () => {
             const props = getBuyProps();
+            const context = getContext({
+                app: {
```

Comment:
why do we have to set this up every time? All these tests can't possibly rely on `stripe`, do they?